docs/documentation/ui_customization/topics/quick-theme.adoc

```diff
@@ -0,0 +1,56 @@
+[[_quick_theme]]
+
+== Customizing with Quick Theme
```

Comment:
Could we publish this also as a guide?

Reply:
@edewit What do you mean?  I envision this document getting published with the rest of our docs.

Is your question specific to Quick Theme?